### PR TITLE
Update Chinese (simplified) translation

### PR DESCRIPTION
**Pull request by womeimingzi11**

### CODE REVIEW

File: MonitorControl/UI/zh-Hans.lproj/Main.strings [hunk 1]
```diff
@@ -1,87 +1,129 @@
...
+"FoA-yh-Yx3.title" = "None";
+
+/* Class = "NSMenuItem"; title = "Minimal"; ObjectID = "Eq3-z9-yIo"; */
+"Eq3-z9-yIo.title" = "Minimal";
+
+/* Class = "NSMenuItem"; title = "Normal"; ObjectID = "Riq-uM-bTs"; */
+"Riq-uM-bTs.title" = "Normal";
+
+/* Class = "NSMenuItem"; title = "Heavy"; ObjectID = "vik-vN-bJe"; */
+"vik-vN-bJe.title" = "Heavy";
```
Comment (JoniVR) [inline MonitorControl/UI/zh-Hans.lproj/Main.strings]: I think you forgot these?

Comment (womeimingzi11) [reply MonitorControl/UI/zh-Hans.lproj/Main.strings]: I have made two more commits to complement these four

**Comment by JoniVR** [comment]
Thanks for implementing the changes so fast! 
I think you still missed one file, look for `Localizable.strings`.

image

**Comment by womeimingzi11** [comment]
> Thanks for implementing the changes so fast!
> I think you still missed one file, look for `Localizable.strings`.
> 
> image

Okay, I have finished them. To be honest, I even don't know there are some strings in the Support Directory, really not familiar with the file structure of Xcode Projects.

Thank you for pointing out this file.

**Comment by JoniVR** [comment]
You missed one more translation, I pushed a commit that added it to the file so all you have to do is fill it in 🙂

image

**Comment by womeimingzi11** [comment]
> You missed one more translation, I pushed a commit that added it to the file so all you have to do is fill it in 🙂
> 
> image

Really really sorry for my carelessness. I have added this missing translation.

Thanks for your check!

**Comment by JoniVR** [comment]
Thanks for the PR @womeimingzi11! 🙂

**Pull request merged**